packages/core/src/Asset/util/isValidHandle.ts

```diff
@@ -1,4 +1,4 @@
 export const isValidHandle = (handle: string) => {
-  const pattern = /^[\w.-]+$/;
+  const pattern = /^[\w.-\s@]+$/;
```

Comment:
Added `\s` as well. Doesn't this means accepting tabs and new lines as well?
I would increase the strength of the tests adding a handle with a space in `invalidHandles` (I guess)

Reply:
you are right, it does support tabs and new lines. Will re do it, and will add to the tests.